.github/workflows/lint.yml

```diff
@@ -76,5 +75,5 @@ jobs:
       - name: Lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          working-directory: extras/dagger/src
+          working-directory: extras/dagger
```

Comment:
Should it go to a separate dagger-related PR?

Reply:
Yes, probably, let me do it. 

The reason I did it here is because I couldn't make this one go green because of the regression.

Reply:
Here it is https://github.com/chainloop-dev/chainloop/pull/604/files